bundles/binding/org.openhab.binding.denon/README.md

```diff
@@ -1,133 +1,140 @@
-## Configuration
+The binding also seems to work with some [Marantz receivers](https://groups.google.com/d/msg/openhab/bSTEfSRt0RU/rOI6uWUe7bIJ).
 
-```
-# denon:<instance>.<property>=value
+<!-- MarkdownTOC depth=1 -->
 
-# IP adress of the Denon receiver instance
-denon:avr2000.host=192.168.1.70
+- [Binding Configuration](#binding-configuration)
+- [Item Configuration](#item-configuration)
+- [Examples](#examples)
+- [References](#references)
 
-# Optional, set connection method for receiving updates. Can be http or telnet. 
-# Denon receivers only support one concurrent telnet connection, so use http if 
-# you have any other app using the telnet connection. Default = telnet
-#denon:avr2000.update=telnet
+<!-- /MarkdownTOC -->
 
-# Optional, this sets the refresh interval (in milliseconds) for all instances 
-# if you're using the http connection method. Default = 5000 
-#denon:refresh=5000
 
-```
-## Items
+## Binding Configuration
+
+The binding can be configured in the file `services/denon.cfg`.
+
+Multiple receivers may be defined. Each requires a unique <name> to be defined.
+
+| Property         | Default | Required | Description                                                 |
+|------------------|---------|----------|-------------------------------------------------------------|
+| &lt;name>.host   |         | Yes      | The IP address of the receiver                              |
+| &lt;name>.update | telnet  | No       | Connection method. Allowed values are "telnet" or "http". Denon receivers only support one concurrent telnet connection, so use http if any other app is using the telnet connection. |
+| refresh          | 5000    | No       | The refresh interval (in milliseconds) for the http connection method. |
+
+
+## Item Configuration
 
-All items use the following format:
+Item bindings must conform to the following format:
 
 ```
-{denon="instance#<property>"}
+{denon="<name>#<property>"}
 ```
 
-| Property | Type | Description | Accepts |
-| :------------- |:-------------| :-----| :-----|
-| **General** |
-| PW | Switch | Main power | On, Off |
-| ZM | Switch | Main zone power | On, Off |
-| Z2 | Switch | Zone 2 power | On, Off |
-| SURROUNDMODE | String | Current surround mode | Read only | 
-| **Volume** |
-| MV | Dimmer | Main volume | Percent, Increase/Decrease |
-| MU | Switch | Mute | On, Off |
-| Z2ZV | Dimmer | Zone 2 volume | Percent, Increase/Decrease |
-| Z2MU | Switch | Zone 2 mute | On, Off |
-| **Input** | 
-| INPUT | String | Select main input | Input name | 
-| SICD | Switch | Switch main input to CD | On |
-| SITV | Switch | Switch main input to TV | On |
-| Z2SOURCE | Switch | Switch zone 2 input to source | On |
-| Z2TUNER | Switch | Switch zone 2 input to tuner | On |
+&lt;name> is the name of the receiver defined in the Binding Configuration, and &lt;property> is one of the following:
```

Comment:
I put all of the `<placeholder>` names in backquotes instead of tricking the Markdown/HTML with the `&lt;` entity because it makes it stand out as something to be replaced.